javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+
+  while (count < (times + 1)) {
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
+
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log(string[i]);
+  }
+}
+
+function doubleAll(numbers) {
+  var doubled = [];
+  for (var i = 0; i < numbers.length; i++) {
+    doubled.push(numbers[i] * 2);
+  }
+  return doubled;
+}
+
+function getKeys(object) {
+  var keys = [];
+  for (var key in object) {
+    keys.push(key);
+  } return keys;
+}
+
+function getValues(object) {
+  var values = [];
+  for (var key in object) {
+    values.push(object[key]);
+  } return values;
+}
```

Comment:
Please do not put `return` statements on the same line as a closing curly brace.

Reply:
<img width="932" alt="Screen Shot 2022-07-09 at 2 39 06 PM" src="https://user-images.githubusercontent.com/97363006/178123551-14c713e6-4ddb-4cfa-931b-99d75d1b55d1.png">